code/game/turfs/open/water.dm

```diff
@@ -359,6 +377,9 @@
 
 /turf/open/water/river/Entered(atom/movable/AM, atom/oldLoc)
 	. = ..()
+	var/mob/living/carbon/human/FM = AM
+	if(isseelie(FM) && !(FM.resting))	//Add wingcheck
```

Comment:
Is the wing check already present here or is the comment indicating a todo?

Reply:
Wing check is not present for any of the water tile movement stuff, but was instead handled by never allowing the Seelie to stand after wing removal. Making it an unnecessary check now.